### PR TITLE
NEXT-21444 - implement banner component

### PR DESCRIPTION
**Pull request by jleifeld**

### CODE REVIEW

**Comment by jleifeld** [comment]
Rename Alert to Banner

File: src/components/base/sw-banner/sw-banner.vue [hunk 1]
```diff
@@ -1,392 +1,435 @@
...
-      default: 'info',
-      validValues: ['info', 'warning', 'error', 'success'],
-      validator(value) {
-        return ['info', 'warning', 'error', 'success'].includes(value);
-      },
+        type: String as PropType<NotificationType>,
+        required: false,
+        default: 'info',
+        validator(value: string): boolean {
+            return ['info', 'warning', 'error', 'success'].includes(value);
```
Comment (seggewiss) [inline src/components/base/sw-banner/sw-banner.vue:73]: Error should be named danger.

**Review by seggewiss — CHANGES_REQUESTED** [review]

File: src/components/base/sw-banner/sw-banner.vue (hunk 1 above)
Comment (jleifeld) [reply src/components/base/sw-banner/sw-banner.vue]: I talked to Dennis. Error is now Critical.

**Review by seggewiss — APPROVED** [review]
LGTM!

**Pull request merged**